mpf/modes/high_score/config/high_score.yaml

```diff
@@ -17,6 +17,13 @@ high_score:
       - HIGH SCORE 2
       - HIGH SCORE 3
       - HIGH SCORE 4
+  defaults:
```

Comment:
@toomanybrians what do you think about the syntax? If you are fine with this we can merge.